src/transformers/convert_slow_tokenizer.py

```diff
@@ -1628,6 +1657,10 @@ def convert_slow_tokenizer(transformer_tokenizer) -> Tokenizer:
 
     tokenizer_class_name = transformer_tokenizer.__class__.__name__
 
+    if tokenizer_class_name in TIKTOKEN_CONVERTERS:
```

Comment:
I think adding a `tiktoken_file` param similar to `gguf_file` would be most consistent. But we actually don't need to pass the file_name via `tiktoken_file`. It would really just be used as a True flag to use tiktoken. Then here in the `convert_slow_tokenizer` function, we can check the flag and the `TIKTOKEN_CONVERTERS`. 

@ArthurZucker  What do you think / is there a better way to pass this flag? I guess since a repo can't have 2 `tokenizer.model` files, we can try to convert via tiktoken first, then sentencepiece. But then it would have to 'error out' on tiktoken to then get to the sentecepiece conversion (most of the models would use spm still).